app/main.py

```diff
@@ -1,8 +1,32 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int, average_rating: int, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, auto: Car) -> float:
+        price = auto.comfort_class * (self.clean_power - auto.clean_mark) * self.average_rating / self.distance_from_city_center
```

Comment:
The calculation of `price` involves division by `self.distance_from_city_center`. Ensure that `distance_from_city_center` is not zero to avoid division by zero errors.